lib/api/manipulation.js

```diff
@@ -363,6 +363,47 @@ exports.wrapInner = _wrap(function (el, elInsertLocation, wrapperDom) {
   updateDOM(wrapperDom, el);
 });
 
+/**
+ * The .unwrap() function, removes the parents of the set of matched elements
+ * from the DOM, leaving the matched elements in their place.
+ *
+ * @example <caption>without selector</caption>
+ *   const $ = cheerio.load(
+ *     '<div id=test>\n  <p>Hello</p></div>\n  <div><p>World</p>\n</div>'
+ *   );
+ *   $('#test p').unwrap();
+ *
+ *   //=> <div id=test>
+ *   //     <p>Hello</p>
+ *   //     <p>World</p>
+ *   //   </div>
+ *
+ * @example <caption>with selector</caption>
+ *   const $ = cheerio.load(
+ *     '<div id=test>\n  <p>Hello</p>\n  <b><p>World</p></b>\n</div>'
+ *   );
+ *   $('#test p').unwrap('b');
+ *
+ *   //=> <div id=test>
+ *   //     <p>Hello</p>
+ *   //     <p>World</p>
+ *   //   </div>
+ *
+ * @param {string} selector - A selector to check the parent element against. If
+ *     an element's parent does not match the selector, the element won't be unwrapped.
+ * @see {@link https://api.jquery.com/unwrap/}
+ * @returns {Cheerio} The instance itself, for chaining.
+ */
+exports.unwrap = function (selector) {
+  var self = this;
+  this.parent(selector)
```

Comment:
I see where the motivation for the parent fix came from 😄

Reply:
true, small but nasty bug